### PR TITLE
implemented paused by guardian

### PR DESCRIPTION
**Pull request by RomanHiden**

### CODE REVIEW

File: contracts/modules/LoanClosings/LoanClosingsWithGasToken.sol [hunk 1]
```diff
@@ -1,119 +0,0 @@
-/**
```
Comment (RomanHiden) [inline contracts/modules/LoanClosings/LoanClosingsWithGasToken.sol:1]: @tomcbean we need to call `replaceContract` with target ZERO to remove this contract. we need to do this so we won't forget

File: contracts/connectors/loantoken/LoanTokenBase.sol [hunk 2]
```diff
@@ -11,10 +11,10 @@ import "../../utils/ReentrancyGuard.sol";
 import "@openzeppelin-2.5.0/ownership/Ownable.sol";
 import "@openzeppelin-2.5.0/utils/Address.sol";
 import "../../interfaces/IWethERC20.sol";
-import "./Pausable.sol";
+import "../../utils/PausableGuardian.sol";
 
 
-contract LoanTokenBase is ReentrancyGuard, Ownable, Pausable {
+contract LoanTokenBase is ReentrancyGuard, Ownable, PausableGuardian {
```
Comment (RomanHiden) [inline contracts/connectors/loantoken/LoanTokenBase.sol:17]: iToken and protocol will have the same `PausableGuardian` style

File: contracts/connectors/loantoken/LoanTokenLogicStandard.sol [hunk 3]
```diff
@@ -174,33 +173,6 @@ contract LoanTokenLogicStandard is AdvancedToken, GasTokenUser {
         );
     }
 
-    function borrowWithGasToken(
```
Comment (RomanHiden) [inline contracts/connectors/loantoken/LoanTokenLogicStandard.sol:177]: completely removed gas token user

File: contracts/governance/PausableGuardian.sol [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        assembly {
+            sstore(slot, isPaused)
+        }
+    }
+
+    function changeGuardian(address newGuardian) public {
+        // getGuardian() == address(0) to allow initial guardian set
+        // current bzxOwner = 0xB7F72028D9b502Dc871C444363a7aC5A52546608
+        // this is to allow initial deploy. later on thru gov vote can be removed once guardians are set.
+        require(getGuardian() == msg.sender || msg.sender == 0xB7F72028D9b502Dc871C444363a7aC5A52546608, "unauthorized");
```
Comment (RomanHiden) [inline contracts/governance/PausableGuardian.sol:43]: this line you may not like, but I don't see other easier way to do so.

File: contracts/connectors/loantoken/LoanTokenLogicStandard.sol [hunk 5]
```diff
@@ -174,33 +175,6 @@ contract LoanTokenLogicStandard is AdvancedToken, GasTokenUser {
         );
     }
```
Comment (tomcbean) [inline contracts/connectors/loantoken/LoanTokenLogicStandard.sol:177]: Functions "borrow" and "marginTrade" are missing pausable. Was this intentional?

**Comment by RomanHiden** [comment]
@tomcbean I ran again tests after your changes everything is passed, unless you have more ideas in mind we can merge this

**Pull request merged**